packages/driver/cypress/e2e/cypress/downloads.cy.ts

```diff
@@ -110,3 +111,24 @@ describe('download behavior', () => {
     cy.readFile(`${Cypress.config('downloadsFolder')}/downloads_does_not_exist.csv`).should('not.exist')
   })
 })
+
+describe('basic auth download behavior', () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/downloads.html', {
+      auth: {
+        username: 'cypress',
```

Comment:
can these values be imported from `cypress/plugins/server.js` to ensure equality?

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/28387/commits/000c18765a31d93b27d441ecb507ae665382eff2